### PR TITLE
identifier tests

### PR DESCRIPTION
**Pull request by daesunp**
## Description

This PR adds the e2e test for identifiers created in [this PR](https://github.com/microsoft/FluidFramework/pull/21506) using the test tree provider.

### CODE REVIEW

File: packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts [hunk 1]
```diff
@@ -1900,6 +1901,42 @@ describe("SharedTree", () => {
 		});
 	});
 
+	describe("Identifiers", () => {
+		it("Can use identifiers and the static Tree Apis", async () => {
```
Comment (noencke) [inline packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts:1905]: This test is under the "Identifiers" describe block but it covers much more than just identifiers - it tests a whole bunch of `Tree.is` functions. The only calls that are actually related to identifiers are the ones to `Tree.shortId`.

I don't remember what's driving this work, but if the intent is only to test identifiers, then you can remove all the other calls. 

Or, just keep them as-is, but rename the describe block or move this test to a different more appropriate describe block somewhere.

Comment (daesunp) [reply packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts:1905]: Yes, the intent was just to test the identifiers. Removed the other calls :)

Comment (noencke) [reply packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts:1905]: Okay, looks better, but you still don't need to be doing the tests of the `at` function, either :)

Comment (noencke) [reply packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts:1905]: I'd just do:

```
const widget = new Widget({});
const fidget = new Widget({ id: "fidget" });
view.initialize([widget, fidget]);
// ... the short ID asserts
```

**Review by noencke — APPROVED** [review]

File: packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts (hunk 1 above)
Comment (daesunp) [reply packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts:1905]: Updated :)

**Pull request merged**